src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/en/strings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "1.0.0.0",
+  "strings": {
+    "name": "Blazor Server App",
+    "description": "A project template for creating a Blazor server app that runs server-side inside an ASP.NET Core app and handles user interactions over a SignalR connection. This template can be used for web apps with rich dynamic user interfaces (UIs)."
```

Comment:
Not positive why my review was requested. But, the English translation looks fine 😃

Reply:
Probably because you have a hook to one of the folders, it looks like GitHub did it automatically. I didn't explicitly add any additional reviewers...